geowebcache/core/src/main/java/org/geowebcache/layer/MetaTile.java

```diff
@@ -315,9 +315,14 @@ public void setImage(RenderedImage metaTiledImage) {
     public RenderedImage createTile(final int minX, final int minY, final int tileWidth,
             final int tileHeight) {
 
+        // optimize if we get a bufferedimage
+        if(metaTileImage instanceof BufferedImage){
+            return ((BufferedImage) metaTileImage).getSubimage(minX, minY, tileWidth, tileHeight);
```

Comment:
Good, it's faster like this. Just to be sure, you tested this with JPEG output, with and without JAI native extensions  (and clearing the cache between runs?). Once upon a time we had issues with the native JPEG writer + images not in the origin, but the image encoding changed quite a bit in the meantime